### ISSUE TITLE
Copying ("Killing") type information at point?

### ISSUE THREAD

**Issue by MarcusRiemer** [ananthakumaran/tide#432]
This might technically be not a tide issue, but my other attempts to solve this issue didn't bring me anywhere. My main problem is that I have lots of functions that have an implicitly defined return type and I need to extract these to explicit type definitions. Everything I want to copy is already there in the area that seems to be the "minibuffer":

![Screenshot of minibuffer with relevant information](https://user-images.githubusercontent.com/2138773/127878998-6b7af214-eca3-4a5e-9456-eaebad890ae2.png)

If I wouldn't know better I would say the machine is mocking me: It undoubtedly has the information I want, but I can't copy it. Clicking it just results in a "Minibuffer window is not active" message. I then tried to ...

* Follow the instructions in this emacs.se question: [How to copy minibuffer contents?](https://emacs.stackexchange.com/questions/15198/how-to-copy-minibuffer-contents)
* Search for an eldoc function like `eldoc-kill-info-at-point`
* Searched for an refactoring extension like "make return type explicit"

In short: I didn't come up with anything helpful and by now I could have solved my problem by taking screenshots and doing some manual OCR. But instead I am now asking you: Is there a way I can somehow geht the type information at point into my kill ring?

Relevant versions: 
- Tide: 20210517.507
- TypeScript: 4.2.4
- Emacs: 27.2

**Comment by ananthakumaran** [comment]
can you set `(setq tide-always-show-documentation t)` and then try `M-x tide-documentation-at-point`. I am assuming this is the same thing we discussed [here](https://github.com/ananthakumaran/tide/discussions/431) last week. Since this question has come up multiple times, this needs to be added to FAQ perhaps

**Comment by MarcusRiemer** [comment]
This indeed solves my problem, thanks alot.

Does `tide-always-show-documentation` only affect `tide-documentation-at-point`? "Customize Group: tide" says:

> Show the documentation window even if only type information is available.

Or to put it another way: What are the downsides from simply keeping that setting permanently enabled?

**Comment by ananthakumaran** [comment]
It affects `tide-documentation-at-point` and company documentation (can be invoked by C-h when completion is shown). The difference is, by default, tide will show the buffer only if there is documentation (not just type information) and will show `"No documentation available."` otherwise. With this flag enabled, it will pop a buffer even only type information is available.
